cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -8,7 +8,7 @@
 import sys
 import requests
 
-from gidgethub.sansio import create_headers
+import gidgethub
```

Comment:
This should either be `import gidgethub.sansio` or `from gidgethub import sansio`; the fact that the package has an import is undocumented.

Reply:
Changed to`from gidgethub import sansio`